pyspider/libs/utils.py

```diff
@@ -223,10 +223,8 @@ def pretty_unicode(string):
     """
     if isinstance(string, six.text_type):
         return string
-    try:
-        return string.decode("utf8")
-    except UnicodeDecodeError:
-        return string.decode('Latin-1').encode('unicode_escape')
+    else:
+        return string.decode("utf8", 'ignore')
```

Comment:
It's not the behavior I wanted. If you are using  `pretty_unicode` on a image for instance, you will get a empty output as the characters would be ignored. But previous version, it will return a string like 'GIF\x01\xsa\x31'

`string.decode('Latin-1').encode('unicode_escape').decode("utf8")` is better